packages/flame/lib/src/sprite.dart

```diff
@@ -130,4 +130,14 @@ class Sprite {
       ..add(image, Vector2.zero(), source: src);
     return composition.compose();
   }
+
+  /// Return a new [Image] based on the [src] of the Sprite.
+  ///
+  /// A sync version of the [toImage] function. Read [Picture.toImageSync] for a
+  /// detailed description of possible benefits in performance.
+  Image toImageSync() {
+    final composition = ImageComposition()
+      ..add(image, Vector2.zero(), source: src);
```

Comment:
The `Vector2.zero` should be possible to add as a static on this class instead, so that it doesn't need to be recreated for every call.

Reply:
Done.